src/cdk/database/serverless-pg.ts

```diff
@@ -1,24 +1,48 @@
+import { ILayerVersion, LayerVersion } from "@aws-cdk/aws-lambda"
 import { DatabaseClusterEngine, IClusterEngine, ServerlessCluster, ServerlessClusterProps } from "@aws-cdk/aws-rds"
 import * as core from "@aws-cdk/core"
+import { Fn, Token } from "@aws-cdk/core"
+import { NodejsFunctionProps } from "@aws-cdk/aws-lambda-nodejs"
+import { IVpc } from "@aws-cdk/aws-ec2"
+
+// default version of https://github.com/jetbridge/lambda-layer-prisma-pg
+export const PRISMA_PG_LAYER_VERSION = 10
+
+let layerVersionCount = 1
 
 export interface SlsPgDbProps extends Omit<ServerlessClusterProps, "engine"> {
   engine?: IClusterEngine
+
+  /**
+   * Number of connections each prisma client can maintain.
+   * Tradeoff of lambdas using up connections vs. query parallelization.
+   * https://www.prisma.io/docs/guides/performance-and-optimization/connection-management#serverless-environments-faas
+   */
+  prismaConnectionLimit?: number
```

Comment:
what is max connections for us?

Reply:
I believe the number is 450. The docs are a bit weak on this point but for normal aurora it would be 450.
2 ACU is 4GB RAM, the formula is here https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/AuroraPostgreSQL.Managing.html#AuroraPostgreSQL.Managing.MaxConnections (for plain Aurora)